services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java

```diff
@@ -29,7 +29,7 @@
   private static final Optional < String > lstn;
   private static final Optional < String > port;
 
-  public static Integer i = new Integer(123);
+  //public static Integer i = new Integer(123);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://178.128.191.195:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)